openapi/node.go

```diff
@@ -23,13 +29,21 @@ type node struct {
 	ref      string
```

Comment:
Your removal of the getRef methods elsewhere rely on this attribute, which is not currently being used for multiple reasons.

1: It doesn't have the appropriate $ref yaml tag
2: Its not accessible to yaml.Unmarshal because its private (check me on this, my reflection might be shaky)
3: Some of the composed types like Schema already have a Ref attribute that has an appropriate tag.

Reply:
it's been removed.